modules/k8s_eks_addons/templates/autoscaler_values.yaml

```diff
@@ -2,20 +2,20 @@ awsRegion: ${aws_region}
 
 autoDiscovery:
   clusterName: ${eks_cluster_id}
+  tags:
+    - k8s.io/cluster-autoscaler/enabled
+    - k8s.io/cluster-autoscaler/${infrastructurename}
+    - k8s.io/cluster-autoscaler/node-template/resources/ephemeral-storage
 extraArgs:
   aws-use-static-instance-list: true
 
 image:
   tag: ${image_tag}
 
-
 resources:
   limits:
     cpu: 200m
     memory: 512Mi
   requests:
     cpu: 200m
     memory: 512Mi
-
-nodeSelector:

```

Comment:
This is removed because it is not used in current state of the reference architecture.
This file previously wasn't used anywhere.
I have aligned to the values file used by eks-addons (.terraform\modules\eks-addons\modules\kubernetes-addons\cluster-autoscaler\values.yaml, after ```terraform init```) and moved it to the modules templates.